extensions/elation/actions/findPhysician.ts

```diff
@@ -0,0 +1,178 @@
+import { ZodError } from 'zod'
+import {
+  FieldType,
+  type Action,
+  type DataPointDefinition,
+  type Field,
+} from '../../../lib/types'
+import { Category } from '../../../lib/types/marketplace'
+import { type settings } from '../settings'
+import { makeAPIClient } from '../client'
+import { fromZodError } from 'zod-validation-error'
+import { AxiosError } from 'axios'
+
+const fields = {
+  firstName: {
+    id: 'firstName',
+    label: 'First Name',
+    description: 'First Name to search for.',
+    type: FieldType.STRING,
+    required: false,
+  },
+  lastName: {
+    id: 'lastName',
+    label: 'Last Name',
+    description: 'Last Name to search for.',
+    type: FieldType.STRING,
+    required: false,
+  },
+  npi: {
+    id: 'npi',
+    label: 'NPI (National Provider Identifier)',
+    description: 'NPI to search for.',
+    type: FieldType.STRING,
+    required: false,
+  },
+} satisfies Record<string, Field>
+
+const dataPoints = {
+  physicianId: {
+    key: 'physicianId',
+    valueType: 'number',
+  },
+  physicianFirstName: {
+    key: 'physicianFirstName',
+    valueType: 'string',
+  },
+  physicianLastName: {
+    key: 'physicianLastName',
+    valueType: 'string',
+  },
+  physicianCredentials: {
+    key: 'physicianCredentials',
+    valueType: 'string',
+  },
+  physicianEmail: {
+    key: 'physicianEmail',
+    valueType: 'string',
+  },
+  physicianNPI: {
+    key: 'physicianNPI',
+    valueType: 'string',
+  },
+  physicianUserId: {
+    key: 'physicianUserId',
+    valueType: 'number',
+  },
+  caregiverPracticeId: {
+    key: 'caregiverPracticeId',
+    valueType: 'number',
+  },
+} satisfies Record<string, DataPointDefinition>
+
+export const findPhysician: Action<
+  typeof fields,
+  typeof settings,
+  keyof typeof dataPoints
+> = {
+  key: 'findPhysician',
+  category: Category.EHR_INTEGRATIONS,
+  title: 'Find Physician',
+  description: "Retrieve a physician using Elation's patient API.",
+  fields,
+  previewable: true,
+  dataPoints,
+  onActivityCreated: async (payload, onComplete, onError): Promise<void> => {
+    try {
+      const { firstName, lastName, npi } = payload.fields
+      // API Call should produce AuthError or something dif.
+      const api = makeAPIClient(payload.settings)
+      const physiciansList = await api.findPhysicians({
+        params: {
+          first_name: firstName,
+          last_name: lastName,
+          npi,
+        },
+      })
+
+      if (physiciansList.count > 1) {
```

Comment:
Should this line be `if (physiciansList.count !== 1)`?

Reply:
Yes, sure 👍🏻